Dockerfiles/trace-agent/stackstate-docker.yaml

```diff
@@ -0,0 +1,15 @@
+## Provides autodetected defaults, for vanilla Docker environments,
+## please see stackstate.yaml.example for all supported options
+
+# Autodiscovery settings for vanilla Docker
```

Comment:
try to remove `listeners` and `config_proveder` they should not be needed

Reply:
removed